Rakefile

```diff
@@ -9,6 +9,7 @@ end
 task :dependencies do
   sh('bundle config set --local path "gems"')
   sh('bundle install')
+  sh('npm install')
```

Comment:
Hrm, I'm not sure I like that `npm` is now a hard-requirement for development. Thoughts on having a separate `test_dependencies`?
Previously we had `validate_site` and `validate_docker` (with `validate` being a meta task), might be nice to do that again so that people who have spelling checked in their editor (there's a VSCode extension for spelling which uses cspell, which for me is a significant reason to pick it over other alternatives) don't need to `npm install`.